Exercises-1-Basic/src/exercises/Number5Shape6.java

```diff
@@ -0,0 +1,27 @@
+package exercises;
+
+/*
+  iii)
+	
+
+     *
+    **
+   ***
+  ****
+ *****
+
+ */
+public class Number5Shape6 {
+
+    public static void main(String[] args) {
+	int number = 1;
+	while (number <= 5) {
+	    for (int i = 1; i <= 5 - number; i++)
```

Comment:
for loop's body should be inside of {} as I asked in other comments as well.

Reply:
Done